src/server/rest/v1/service/TransactionService.ts

```diff
@@ -40,6 +40,28 @@ import moment from 'moment-timezone';
 const MODULE_NAME = 'TransactionService';
 
 export default class TransactionService {
+
+  public static async handleGetTransactions(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<void> {
+    const filteredRequest = TransactionSecurity.filterGetTransactions(req.query);
```

Comment:
Call right await the getTransactions() with the proper fields projection (related to my other comments below)

Reply:
Done